client/config/toml.go

```diff
@@ -27,6 +30,17 @@ node = "{{ .Node }}"
 broadcast-mode = "{{ .BroadcastMode }}"
 `
 
+	defaultHomeTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                            Home Configuration                             ###
+###############################################################################
+
+# The currently configured home directory
+home-dir = "{{ .HomeDir }}"`
```

Comment:
**Note:** This unfortunately MUST be something other than just `home` because when unpacking using the toml parser this apparently returns the value behind `$HOME`, which took me quite some time to figure out why the hell this wouldn't work..